sdk/openai/tests.yml

```diff
@@ -24,24 +22,3 @@ stages:
         AZURE_OPENAI_SEARCH_KEY: $(COGNITIVE-SEARCH-API-KEY)
         AZURE_OPENAI_SEARCH_INDEX: $(COGNITIVE-SEARCH-API-INDEX)
         AZURE_TEST_RUN_LIVE: 'true'
-  - template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
```

Comment:
Side note, this simplification means that we have I have one less `tests.yml` to special case in the `1es` transition 👍 👍 👍